bundles/io/org.openhab.io.caldav/src/main/java/org/openhab/io/caldav/CalDavQuery.java

```diff
@@ -27,6 +27,7 @@
     private Sort sort;
     private String filterName;
     private List<String> filterCategory;
+    private boolean filterCategoryMatchesAny = false;
```

Comment:
yes, to keep compatibility with previous configurations. I made it so that by default a query works as before, if not specifically modified with a "filter-category-any". I wanted to be sure to not interfere with previous code